Container Security/eks-implementation-guide.md

```diff
@@ -22,14 +22,21 @@ The Falcon Container sensor for Linux extends runtime security to container work
 
 ## Pre-requisites
 
+- Existing AWS Account and VPC
+- You will need a workstation with a linux platform
```

Comment:
So windows and OSX-based system administrators cannot deploy LUMOS? If that's accurate, can we detail why?